src/tcgwars/logic/groovy/TcgStatics.groovy

```diff
@@ -485,20 +485,16 @@ class TcgStatics {
       opp.bench.remove(pcs)
     }
   }
-  static PokemonCardSet benchPCS (Card card, ActivationReason reason=ActivationReason.OTHER, TargetPlayer targetPlayer=TargetPlayer.SELF){
+  static PokemonCardSet benchPCS (Card card, ActivationReason reason=OTHER, TargetPlayer targetPlayer=TargetPlayer.SELF){
```

Comment:
targetPlayer parameter should be removed, along with all usages

Reply:
Removed third parameter for TargetPlayer and all usages